tests/api/controllers/supervisor_import_remove_mf/supervisor_import_remove_mf.c

```diff
@@ -38,49 +38,55 @@ int main(int argc, char **argv) {
   // Import
 
   wb_robot_step(TIME_STEP);
-
   wb_supervisor_field_insert_mf_bool(bool_field, 0, true);
   reference_distance -= 0.05;
   wb_robot_step(TIME_STEP);
   ts_assert_double_in_delta(wb_distance_sensor_get_value(distance_sensor), reference_distance, 0.0001,
                             "Distance after insertion of MFBool incorrect.");
 
+  wb_robot_step(TIME_STEP);
```

Comment:
Are the calls to wb_robot_step(TIME_STEP) that you've added to this file related to this PR in some way? If they are, please explain. If they aren't, please move them to another PR if they are needed to address some other issue.

Reply:
Nevermind. I understand now.